media-fonts/cascadia-code/cascadia-code-2102.03.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+FONT_SUFFIX="ttf"
+inherit font
+
+DESCRIPTION="A monospaced font developed for Windows Terminal"
+HOMEPAGE="https://github.com/microsoft/cascadia-code"
+SRC_URI="https://github.com/microsoft/cascadia-code/releases/download/v${PV}/CascadiaCode-${PV}.zip -> ${P}.zip"
```

Comment:
Do they supply non-zips, allowing us to drip the BDEPEND?

Reply:
Unfortunately not for the compiled font files. There is a source tarball, but I already looked into compiling this font some time ago and it would require quite some dependencies.